code/modules/antagonists/spy/spy.dm

```diff
@@ -130,6 +130,30 @@
 		your_mission.owner = owner
 		your_mission.explanation_text = pick_list_replacements(SPY_OBJECTIVE_FILE, "objective_body")
 		objectives += your_mission
+	
+	if(LAZYLEN(objectives) < 3)
+		if(prob(25))
```

Comment:
`if(LAZYLEN(objectives) < 3 && prob(25))`. I can't seem to suggest changes on this, thanks github. 
Also, it should be `length(objectives)`. Fairly certain it's not a lazylist.

Reply:
```suggestion
	if(LAZYLEN(objectives) < 3 && prob(25))
```
Unindent the rest, etc